docs/website/docs/dlt-ecosystem/verified-sources/mongodb.md

```diff
@@ -0,0 +1,384 @@
+resource for each item in collection_list.
+
+### Source `mongo_collection`
+
+This function fetches data from a MongoDB collection, returning resources for each collection to be loaded.
+
+```python
+def mongodb_collection(
+    connection_url: str = dlt.secrets.value,
+    database: Optional[str] = dlt.config.value,
+    collection: str = dlt.config.value,
+    incremental: Optional[dlt.sources.incremental] = None,  # type: ignore[type-arg]
+    write_disposition: Optional[str] = dlt.config.value,
+) -> Any:
+```
+
+Args:
+
+The arguments `connection_url`, `database`, `incremental` and `write_disposition` are the same as
+defined for [Source `mongodb`](#source-mongodb).
+
+`collection`: Name of the collection to load.
+
+Returns:
+
+Iterable [DltResource]: A list of DLT resources ("collection_documents") for each collection to be
+loaded.
+
+### Resource `collection_obj.name`
+
+This function, part of the "mongo_collection" source, loads documents from a collection. Each
+resource is invoked with the MongoDB client, collection object, and incremental parameter, preparing
+a DLT resource for each item in the collection.
+
+```python
+return dlt.resource(  # type: ignore
+    collection_documents,
+    name=collection_obj.name,
+    primary_key="_id",
+    write_disposition=write_disposition,
+)(client, collection_obj, incremental=incremental)
+```
+
+`name`: Collection object's name.
+
+Parameters `primary_key` and `write_disposition` are as mentioned in
+[resource `collection_documents`.](#resource-collectiondocuments)
+
+### Create your own pipeline
+
+If you wish to create your own pipelines, you can leverage source and resource methods from this
+verified source.
+
+1. Configure the pipeline by specifying the pipeline name, destination, and dataset as follows:
+
+   ```python
+   pipeline = dlt.pipeline(
+        pipeline_name="mongodb_pipeline",  # Use a custom name if desired
+        destination="duckdb",  # Choose the appropriate destination (e.g., duckdb, redshift, post)
+        dataset_name="mongodb_data"  # Use a custom name if desired
+   )
+   ```
+
+1. To load all the collections in a database:
+
+   ```python
+   load_data = mongodb()
+   load_info = pipeline.run(load_data, write_disposition="replace")
+   print(load_info)
+   ```
+
+1. To load a specific collections from the database:
+
+   ```python
+   load_data = mongodb().with_resource("collection_1", "collection_2")
+   load_info = pipeline.run(load_data, write_disposition="replace")
+   print(load_info)
+   ```
+
+1. To load specific collections from the source incrementally:
+
+   ```python
+   load_data = mongodb(incremental=dlt.sources.incremental("date")).with_resources("collection_1")
+   load_info = pipeline.run(load_data, write_disposition = "merge")
+   print(load_info)
+   ```
+   > Data is loaded incrementally based on "date" field.
+
+1. To load data from a particular collection say "movies" incrementally:
+
+   ```python
+   load_data = mongodb_collection(
```

Comment:
we need to explain why we used mongodb as a source above, and now we use mongodb_collection. what is the difference between them?

Reply:
added context